.gitignore

```diff
@@ -71,7 +71,12 @@ packages/*/dist
 packages/generator/renderedTemplates/
 .vscode
 
-**/*.js
 reports*
 
 .DS_Store
+
+# Local History for Visual Studio
+**/.history
+
+#Not committing this, not needed and causes issues with circleCI
```

Comment:
[NIT] Not totally accurate...it will actually cause issues anytime you ran the tests and there was a change to anything in core or the sdk.